src/runtime/pkg/resourcecontrol/utils_linux_test.go

```diff
@@ -102,74 +102,7 @@ func TestValidCgroupPathV1(t *testing.T) {
 		} else {
 			assert.True(
 				strings.HasPrefix(path, DefaultResourceControllerID),
-				"%v should have prefix /%v", path, DefaultResourceControllerID)
-		}
-	}
-
-}
-
-func TestValidCgroupPathV2(t *testing.T) {
```

Comment:
When removing the old code's UT, please add new UTs for the added code as well.

Reply:
`ValidCgroupPathV1()` and `ValidCgroupPathV2()` were merged into `ValidCgroupPath()`.